src/ui/views/LandingPages/Welcome/AddressImport/SetPassword.tsx

```diff
@@ -0,0 +1,240 @@
+import { Box, Typography, FormGroup } from '@mui/material';
+import { makeStyles } from '@mui/styles';
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { storage } from '@/background/webapi';
+import { getHashAlgo, getSignAlgo } from '@/shared/utils/algo';
+import {
+  PasswordInput,
+  TermsCheckbox,
+  ErrorSnackbar,
+  SubmitButton,
+} from '@/ui/FRWComponent/LandingPages/SetPassword';
+import SlideRelative from '@/ui/FRWComponent/SlideRelative';
+import { useWallet, saveIndex } from 'ui/utils';
+
+import CheckCircleIcon from '../../../../../components/iconfont/IconCheckmark';
+import CancelIcon from '../../../../../components/iconfont/IconClose';
+
+const useStyles = makeStyles(() => ({
+  inputBox: {
+    height: '64px',
+    padding: '16px',
+    zIndex: '999',
+    backgroundColor: '#282828',
+    border: '2px solid #4C4C4C',
+    borderRadius: '12px',
+    boxSizing: 'border-box',
+    '&.Mui-focused': {
+      border: '2px solid #FAFAFA',
+      boxShadow: '0px 8px 12px 4px rgba(76, 76, 76, 0.24)',
+    },
+  },
+}));
+
+const SetPassword = ({ handleClick, mnemonic, pk, username, tempPassword, accounts, goEnd }) => {
```

Comment:
This should be a reusable component. It should only be used once

Reply:
-